Makefile

```diff
@@ -196,7 +227,7 @@ distclean: clean
 	rm -f Makefile.winsdk
 
 clean:
-	rm -f *.obj *.o *.lib *.a *.exe *.opt *.cmx *.dll *.exp *.cmi *.cmo *~ version.res version.ml
+	rm -f *.obj *.o *.lib *.a *.exe *.opt *.cmx *.dll *.exp *.cmi *.cmo *~ version.res version.ml COMPILER-* cmdliner.ml BytesCompat.ml StringCompat.ml
```

Comment:
Typo cmdliner.ml (extra 'r'), although this would go away according to the previous note.